libsplinter/src/peer/connector.rs

```diff
@@ -288,6 +288,12 @@ impl PeerRemover {
             .map_err(|err| PeerRefRemoveError::ReceiveError(format!("{:?}", err)))?
     }
 
+    /// This function will only be called when the `EndpointPeerRef` is dropped.
+    ///
+    /// Sends a request to the `PeerManager` to remove a peer.
```

Comment:
I suggest swapping the order of these two sentences, so that the description starts with info on what it does (which would make it consistent with other function descriptions).